src/modules/site-v2/base/views/admin/etl.py

```diff
@@ -94,21 +97,18 @@ def create_op():
   ## POST Request ##
   # Submit the form
 
-  # Extract form fields
-  db_op = request.form.get('db_op')
-  note = request.form.get('note')
   user = get_current_user()
-  args = {
-    'SPECIES_LIST': form.data.get('species'),
-  }
-
-  # Pre-flight check: Ensure all necessary files exist in the database
-  missing_files = db_op_preflight_check(db_op, args['SPECIES_LIST'])
-  if len(missing_files) > 0:
-    files_txt = ''.join([ f'<br />{filename}' for filename in missing_files ])
+
+  # Try parsing the form into a job handler & submitting the job
+  try:
+    job = DatabaseOperationPipeline.create(user, form.data)
```

Comment:
I've reanalyzed the `db_op_preflight_check` code as part of the parsing step for a `DatabaseOperationPipeline` job, so the code has been moved to that class (i.e. no need to explicitly call a check here).